common/metrics/metric_defs.go

```diff
@@ -1492,39 +1492,39 @@ var (
 	// Matching
 	MatchingClientForwardedCounter            = NewCounterDef("forwarded")
 	MatchingClientInvalidTaskQueueName        = NewCounterDef("invalid_task_queue_name")
-	SyncMatchLatencyPerTaskQueue              = NewRollupTimerDef("syncmatch_latency_per_tl", "syncmatch_latency")
-	AsyncMatchLatencyPerTaskQueue             = NewRollupTimerDef("asyncmatch_latency_per_tl", "asyncmatch_latency")
-	PollSuccessPerTaskQueueCounter            = NewRollupCounterDef("poll_success_per_tl", "poll_success")
-	PollTimeoutPerTaskQueueCounter            = NewRollupCounterDef("poll_timeouts_per_tl", "poll_timeouts")
-	PollSuccessWithSyncPerTaskQueueCounter    = NewRollupCounterDef("poll_success_sync_per_tl", "poll_success_sync")
-	LeaseRequestPerTaskQueueCounter           = NewRollupCounterDef("lease_requests_per_tl", "lease_requests")
-	LeaseFailurePerTaskQueueCounter           = NewRollupCounterDef("lease_failures_per_tl", "lease_failures")
-	ConditionFailedErrorPerTaskQueueCounter   = NewRollupCounterDef("condition_failed_errors_per_tl", "condition_failed_errors")
-	RespondQueryTaskFailedPerTaskQueueCounter = NewRollupCounterDef("respond_query_failed_per_tl", "respond_query_failed")
-	SyncThrottlePerTaskQueueCounter           = NewRollupCounterDef("sync_throttle_count_per_tl", "sync_throttle_count")
-	BufferThrottlePerTaskQueueCounter         = NewRollupCounterDef("buffer_throttle_count_per_tl", "buffer_throttle_count")
-	ExpiredTasksPerTaskQueueCounter           = NewRollupCounterDef("tasks_expired_per_tl", "tasks_expired")
+	SyncMatchLatencyPerTaskQueue              = NewTimerDef("syncmatch_latency")
+	AsyncMatchLatencyPerTaskQueue             = NewTimerDef("asyncmatch_latency")
+	PollSuccessPerTaskQueueCounter            = NewCounterDef("poll_success")
+	PollTimeoutPerTaskQueueCounter            = NewCounterDef("poll_timeouts")
+	PollSuccessWithSyncPerTaskQueueCounter    = NewCounterDef("poll_success_sync")
+	LeaseRequestPerTaskQueueCounter           = NewCounterDef("lease_requests")
+	LeaseFailurePerTaskQueueCounter           = NewCounterDef("lease_failures")
+	ConditionFailedErrorPerTaskQueueCounter   = NewCounterDef("condition_failed_errors")
+	RespondQueryTaskFailedPerTaskQueueCounter = NewCounterDef("respond_query_failed")
+	SyncThrottlePerTaskQueueCounter           = NewCounterDef("sync_throttle_count")
+	BufferThrottlePerTaskQueueCounter         = NewCounterDef("buffer_throttle_count")
+	ExpiredTasksPerTaskQueueCounter           = NewCounterDef("tasks_expired")
 	ForwardedPerTaskQueueCounter              = NewCounterDef("forwarded_per_tl")
```

Comment:
This `per_tl` metric is still needed?

Reply:
My understanding is the per_tl metrics will be deprecated. @yiminc to confirm.

Reply:
correct, deprecate per_tl metrics.